src/config/dynamic-theme-fixes.config

```diff
@@ -575,6 +575,18 @@ INVERT
 
 ================================
 
+bbs.chinauos.com
+
+INVERT
+.post_tip
+
+CSS
+div[style="all: initial;"] {
```

Comment:
Where can I find this element?

Reply:
It's in the link I pasted above: https://bbs.chinauos.com/post/1930?id=1930&type_id=0

Reply:
Please use `^=` So it would match with elements that begin with that text. As dark reader is adding all their inline nonsense styles into it.

Reply:
Updated :)